manage_breast_screening/assets/sass/main.scss

```diff
@@ -33,3 +33,15 @@ Highlight in red links that don't get anywhere yet
 a[href="#"] {
   color: #c80000;
 }
+
+// Status tag positioning within headers
+.app-header-with-status {
```

Comment:
should this be named something less specific than `with-status`?

Reply:
agree, will rename to `.app-header` (along with the status tag rule below it)